### DISCUSSION TITLE
机器人不能回复自己消息

### DISCUSSION THREAD

**Discussion by snowy-yang** [nonebot/discussions#460] [问答]
![3fcb2a8958b121cfc7cecba876db8e3d](https://github.com/nonebot/discussions/assets/102860110/dce6eb4b-a101-4f5e-975d-338fc746f8d4)

**Comment by yanyongyu** [comment]
如你所见，机器人自己的消息事件类型为`message_sent`，那自然不会被`on_message`等消息响应器响应。另外你的`message_sent`类型的响应器成功响应了但并未有回复的话，建议你打开debug日志，应该是类型不一致触发重载跳过了。

**Reply by snowy-yang** [reply]
怎么开debug

**Reply by yanyongyu** [reply]
参考文档配置部分修改`log_level`配置项

**Reply by snowy-yang** [reply]
![eeeeb322061a5783dbfb2d6882130933](https://github.com/nonebot/discussions/assets/102860110/4f93cce1-99d1-4a5f-96d7-2dd487c679f8)

**Comment by yanyongyu** [comment]
你的handler类型标注有问题导致被跳过，请检查你的handler。

<img width="834" alt="image" src="https://github.com/nonebot/discussions/assets/42488585/1a662be4-0d0c-4266-9766-a89ea31e50cb">
